monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -67,6 +70,8 @@ def technique_status(cls):
                                           'data.status': ScanStatus.SCANNED.value,
                                           'data.technique': cls.tech_id}):
             return ScanStatus.SCANNED.value
+        elif not AttackConfig.get_technique_values()[cls.tech_id]:
+            return ScanStatus.DISABLED.value
```

Comment:
1. We can move this if to be the first if, that way it'll save 2 Mongo calls sometimes
2. Perhaps extract the conditional into a separate function which is well named (and easier to test)?